src/coreclr/tools/aot/ILCompiler/ILCompilerRootCommand.cs

```diff
@@ -262,8 +262,9 @@ public ILCompilerRootCommand(string[] args) : base(".NET Native IL Compiler")
                         // + the original command line arguments
                         // + a rsp file that should work to directly run out of the zip file
 
-                        Helpers.MakeReproPackage(makeReproPath, context.ParseResult.GetValue(OutputFilePath), args,
-                            context.ParseResult, new[] { "r", "reference", "m", "mibc", "rdxml", "directpinvokelist", "descriptor" });
+                        Helpers.MakeReproPackage(makeReproPath, context.ParseResult.GetValue(OutputFilePath), args, context.ParseResult,
+                            inputOptions : new[] { "r", "reference", "m", "mibc", "rdxml", "directpinvokelist", "descriptor", "root", "conditionalroot", "trim" },
```

Comment:
The `root`, `conditionalroot` and `trim` options might not be actual files, it could be just an assembly name. Also these must be a subset of `r` on input files so we'll capture the files for sure. I don't think we should include them here.

Reply:
I understand. 
The problem comes if one of these options actually contains a full path, would it make sense that in such cases the repro package response file just includes the names of assemblies instead. For example:
| original.rsp | repro.rsp |
| --- | --- |
| --root:<full_path>AssemblyName.dll | --root:AssemblyName |

Reply:
The repro should rewrite all absolute paths with relative paths, do in this case would expect the `repro.rsp` to contain `--root:<relativePath>/AssemblyName.dll`

Reply:
I think Michal was suggesting that if in the `original.rsp` there is: `--root:AssemblyName`, then my changes will not work. 
So the code needs to detect:
- if the absolute path was used, convert it to relative
- if the assembly name is used, pass along  
I will look into that.